android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/screens/MeasureReportResultScreen.kt

```diff
@@ -61,17 +61,16 @@ import org.smartregister.fhircore.quest.ui.shared.models.MeasureReportPatientVie
 
 @Composable
 fun MeasureReportResultScreen(
-  reportId: String,
   navController: NavController,
   measureReportViewModel: MeasureReportViewModel
 ) {
   val uiState = measureReportViewModel.reportTypeSelectorUiState.value
 
   // Previously selected measure from the list of supported measures
-  val measureReportRowData = measureReportViewModel.measureReportConfig.value
+  val measureReportRowData = measureReportViewModel.measureReportConfigList
 
   MeasureReportResultPage(
-    screenTitle = measureReportRowData?.title ?: "",
+    screenTitle = measureReportRowData.first().module,
```

Comment:
Using the extension `first()` on a List will cause app crash with `NoSuchElementException`. It will be safe to check if the list is empty first or use `firstOrNull()` extension instead.
```suggestion
    screenTitle = measureReportRowData.firstOrNull().module ?: "",
```